packages/@sanity/desk-tool/src/pane/Editor.js

```diff
@@ -305,10 +305,11 @@ export default withRouterHOC(
     handleCreateCopy = () => {
       const {router, draft, published, paneIndex} = this.props
       const prevId = getPublishedId((draft || published)._id)
+      const omit = ['_createdAt', '_updatedAt']
 
       const duplicatedDocument = this.isLiveEditEnabled()
-        ? copyDocument(published)
-        : newDraftFrom(copyDocument(draft || published))
+        ? copyDocument(published, {omit})
+        : newDraftFrom(copyDocument(draft || published, {omit}))
```

Comment:
Should we also omit `_createdAt` and `_updatedAt` when moving from doc to draft?

Reply:
Not saying we should/shouldn't, but unsure if I can imagine all use cases from where I'm sitting right now

Reply:
This is only when explicitly using the "duplicate document" action - the check highlighted here just checks whether or not we're in "live edit" mode or not.

When editing a document, I think it makes sense to keep the timestamps. When _duplicating_ a document, I feel like that should be treated as a new document but with the same... "user-level" field values. My two cents anyway :)

Reply:
Gotcha. Then I'm all for this! Good job Espen 👍